src/common/schemas/input/get-balances.json

```diff
@@ -11,9 +11,9 @@
     "options": {
       "description": "Options to filter and determine which balances to return.",
       "properties": {
-        "counterparty": {
```

Comment:
To make this PR smaller/easier to read, don't make this change outside of the actual sugar methods - we're going to strip out jsonschemas anyways.

Reply:
Deleted!

Reply:
You can just remove all validation calls and use the new `verify` methods instead. Don't use anything jsonschemas-related.